text/0000-collocate-in-repo-addon-tests.md

```diff
@@ -0,0 +1,48 @@
+- Start Date: 2018-09-05
+- RFC PR: (leave this empty)
+- Ember Issue: (leave this empty)
+
+# Collocate in-repo-addon tests
+
+## Summary
+
+Locate tests that are associated with an in-repo-addon in that addon's tests folder. However, for this RFC, the output of the tests bundles will remain identical to the current output.
+
+## Motivation
+
+Tests that are collocated next to their code in addons will be more discoverable and maintainable.
+
+Having a simple syntax for executing tests associated with an addon is useful and time-saving.
+
+## Detailed design
+
+- Indicate that an addon has tests that are collocated
+  - use `includeTestsInHost: true` in `index.js` to indicate collocated tests
+- Implement CLI flag that only runs the tests for a specified addon
+  - `ember test --ir-addon foo-bar`
+  - The flag's use would create a list of tests to run that are collocated in that in-repo-addon's test folder. The tests will still be included all in one bundle, so this is basically just running the modules that are associated with the co-located tests.
```

Comment:
Since all the test are included into a single bundle why do we need `--ir-addon` flag? Build takes a while, building several `--ir` tests may be annoying. 

What if we provide something like _"Engines"_ or _"Bundles"_ dropdown in qunit interface?